### PR TITLE
spec for create file share

### PR DESCRIPTION
**Pull request by gnehapk**

### CODE REVIEW

File: specs/create-file-share-spec.adoc [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+=== Impacted Modules:
+
+==== Tendrl API impact:
+
+    * API to create file share - Required
+
+    * API to get list of bricks - Required
+
+    * API to create brick - Required
```
Comment (gnehapk) [inline specs/create-file-share-spec.adoc:39]: @anivargi Please provide the mentioned APIs.

**Comment by gnehapk** [comment]
@anivargi @shtripat @nthomas-redhat @kamleshverma1 @cloudbehl  @nnDarshan  Please review. I have mentioned only the things that we need from API in UI. Will detail this spec later on.

@nnDarshan @shtripat  Which access protocols are supported? Do we have cache options and Thread options(https://redhat.invisionapp.com/share/Q78YMAVDJ#/screens/197076159) data available which needs to be shown on the UI

@anivargi Will it be possible to implement Users and permission in API as we don’t have support for user management in Tendrl? It is required in UX design- https://redhat.invisionapp.com/share/Q78YMAVDJ#/screens/197076156

**Comment by nnDarshan** [comment]
@gnehapk gluster is the access protocol thats supported for now.

Regarding cache options that is not available for now...even the UX design page mentions that need not be part of mvp. So we can add that later

File: specs/create-file-share-spec.adoc (hunk 1 above)
Comment (gnehapk) [reply specs/create-file-share-spec.adoc:39]: @anivargi Please provide the API for getting list of disks as well.

**Comment by r0h4n** [comment]
@gnehapk Is this done?

**Pull request closed without merging**